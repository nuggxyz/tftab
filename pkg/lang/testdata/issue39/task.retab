
func hello_world {
	params = [a]
	result = "hello, ${a}!"
}


gen taskfile {
	// schema = "https://taskfile.dev/schema.json"
	path = "../taskfile.yaml"
	data = {
		version = 3
		tasks = {
			abc = {
				cmd = hello_world( "world")
			}
		}
	}
}
