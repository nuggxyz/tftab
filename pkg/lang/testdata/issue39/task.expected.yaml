# code generated by retab. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

version: 3
tasks:
    abc:
        cmd: hello, world!
